core/tests/build_sources/templates/base.html

```diff
@@ -217,7 +217,7 @@ <h2>
     <script src="/templates/dev/head/components/SkillCreationService.js"></script>
     <script src="/templates/dev/head/components/StoryCreationService.js"></script>
 
-    <script src="/templates/dev/head/components/CkEditorRteDirective.js"></script>
+    <script src="/templates/dev/head/components/ck-editor-helper/ck-editor-4.directive.ts"></script>
```

Comment:
Ditto.

Reply:
same here @YashJipkate or you dint change it for a reason?

Reply:
Files in build_sources aren't served either.